exercises/nucleotide-count/nucleotide_count_test.go

```diff
@@ -79,6 +83,10 @@ var histogramTests = []histogramTest{
 		Histogram{'A': 0, 'C': 0, 'T': 0, 'G': 8},
 	},
 	{
+		"GGXXGGGG", // skip mutation
```

Comment:
Removed this test case. 

I was planning to open another after this one for cleaning as suggested by @kytrinyx [here](https://github.com/exercism/xgo/pull/407#issuecomment-255615703). I can include error and test case there.